cmds/webboot/webboot_test.go

```diff
@@ -105,3 +109,98 @@ func TestDownloadByBookmarkOption(t *testing.T) {
 		t.Errorf("Fail to remove test file, error msg: %+v", err)
 	}
 }
+
+func TestInstallCachedISOOption(t *testing.T) {
+	for _, tt := range []struct {
+		name      string
+		userInput []string
+	}{
+		{
+			name:      "hit first iso in the first dir",
+			userInput: []string{"0", "<Enter>", "0", "<Enter>"},
+		},
+		{
+			name:      "hit second iso in the second dir",
+			userInput: []string{"1", "<Enter>", "1", "<Enter>"},
+		},
+	} {
+		t.Run(tt.name, func(t *testing.T) {
+			uiEvents := make(chan ui.Event)
+			go pressKey(uiEvents, tt.userInput)
+
+			var entry menu.Entry = &DirGroup{uiEvents: uiEvents, path: "./testdata/", name: "test install cached iso option"}
```

Comment:
Since this is just a test, you don't need to provide a name in this struct, it'll default to ""